integrations/github/github/core/exporters/pull_request_exporter.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE, RAW_ITEM
+from loguru import logger
+from github.core.options import SinglePullRequestOptions, ListPullRequestOptions
+from github.core.exporters.abstract_exporter import AbstractGithubExporter
+from github.clients.http.rest_client import GithubRestClient
+
+
+class RestPullRequestExporter(AbstractGithubExporter[GithubRestClient]):
+
+    async def get_resource[
+        ExporterOptionsT: SinglePullRequestOptions
+    ](self, options: ExporterOptionsT,) -> RAW_ITEM:
+        repo_name = options["repo_name"]
+        pr_number = options["pr_number"]
+
+        endpoint = f"{self.client.base_url}/repos/{self.client.organization}/{repo_name}/pulls/{pr_number}"
+        response = await self.client.send_api_request(endpoint)
+
+        logger.debug(f"Fetched pull request with identifier: {repo_name}/{pr_number}")
+
+        response["__repository"] = repo_name
```

Comment:
can we maybe add a helper method, that will do those resource enrichments?

Reply:
that way if in the future we are adding a new method or something we won't need to remember how we are enriching